observations/README.md

```diff
@@ -0,0 +1,1279 @@
+| identifier\[0].system                               | `"https://PSSAdaptor/{{losingOdsCode}}"` - where the `{{losingOdsCode}}` is the ODS code of the losing practice                                        |
+| identifier\[0].value                                | `CompoundStatement / id \[@root]`                                                                                                                      |
+| status                                              | fixed value = `"final"`                                                                                                                                |
+| code                                                | `CompoundStatement / code` as described in the XML > FHIR section of [Codeable Concept](../codeable%20concept/README.md)                               | 
+| subject                                             | reference to the mapped [Patient](../patient/README.md)                                                                                                |
+| context                                             | reference to the associated [Encounter](../encounters/README.md) (if present)                                                                          |
+| effectiveDateTime                                   | `CompoundStatement / effectiveTime` or else `CompoundStatement / availibiltyTime [@value]`                                                             |
+| issued                                              | `ehrCompostion / author / time [@value]` <sup>2</sup> or else `EhrExtract / availibilityTime [@value]`                                                 | 
+| performer\[0]                                       | Practitioner referenced in `CompoundStatement / paticipant` <sup>3</sup>                                                                               |
+| component\[index].code                              | `ObservationStatement.code` <sup>4</sup>   as described in the XML > FHIR section of [Codeable Concept](../codeable%20concept/README.md)               |
+| component\[index].valueQuantity                     | `ObservationStatement / value` where `ObservationStatement / value [@type]` is `"PQ"` or `"IVLPQ"` <sup>4</sup>                                        |
+| component\[index].valueQuantity.extension\[0].url   | fixed value = `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ValueApproximation-1"` <sup>1</sup>                             |
+| component\[index].valueQuantity.extension\[0].value | fixed value = `true` <sup>1</sup>                                                                                                                      |
+| comment                                             | concatenated from `ObservationStatement / pertinentInformation / pertinentAnnotation / text` <sup>5</sup> and `NarrativeStatement / text` <sup>6</sup> |
```

Comment:
Another example of https://github.com/NHSDigital/patient-switching-adaptors-mapping-documentation/pull/13/files#r1238515661

Reply:
Although, actually do the footnotes cover it here 🤔

Reply:
Partially, I've added "each" again. And updated the footnote, as they're only added if they have an appropriate SNOMED code